draft-ietf-rats-architecture.md

```diff
@@ -69,27 +69,113 @@ remote attestation procedures (RATS).
 
 # Terminology {#terminology}
 
-    <this section can include content from Terminology in draft-birkholz-rats-architecture
-    and draft-thaler-rats-architecture>
+This document uses the following terms:
+
+* Appraisal Policy: A set of rules that direct how a system evaluates
+    the validity of information about another entity.  Compare /security policy/
+    in {{?RFC4949}}.
+
+* Attestation: A process by which one entity (the "Attester") provides evidence
+    about its identity and state to another entity, which then assesses its
+    trustworthiness.
+
+* Attestation Result: The evaluation results generated by a Verifier,
+    typically including information about an Attester, where the
+    Verifier vouches for the validity of the results.
+
+* Attester: An entity whose attributes must be evaluated in order to
+    determine whether the entity is considered trustworthy, such as when
+    deciding whether the entity is authorized to perform some operation.
+
+* Endorsement: A secure statement that some entity (typically a manufacturer) vouches
+    for the integrity of an Attester's signing capability.
```

Comment:
Does the Known-Good-Value/Reference Value fall into Endorsement or Appraisal Policy? Is it better to add some text into the explanation of the related term to explicitly express that this term includes the Reference Value?

Reply:
Appraisal Policy! (No, could be both)   Reference Value is a 1:1 comparison.   But, for geo-coordinates it would be more like a policy, a calculation is required.  Appraisal Policy is a super set of Reference Value/KGV. An Endorsement is spoken by a supply chain entity, while an Appraisal Policy is spoken by an Owner.

Reply:
Appraisal Policy.   And as Michael implies, Reference Value is just an example and is not sufficient in the general case so we do not want to imply that there is necessarily such a thing as a Reference Value, except in a specific example.

Reply:
Got it. Use Cases draft or 'Use Cases' section is a good place for giving an example to explain that Reference Value is one case of Appraisal Policy.